corehq/apps/userreports/pillow.py

```diff
@@ -194,14 +194,16 @@ def _rebuild_es_tables(self, adapters):
             adapter.rebuild_table_if_necessary()
 
     def rebuild_table(self, adapter):
-        config = adapter.config
-        if not config.is_static:
-            latest_rev = config.get_db().get_rev(config._id)
-            if config._rev != latest_rev:
-                raise StaleRebuildError('Tried to rebuild a stale table ({})! Ignoring...'.format(config))
-        adapter.rebuild_table()
-        if self.auto_repopulate_tables:
-            rebuild_indicators.delay(adapter.config.get_id)
+        # https://manage.dimagi.com/default.asp?252832
+        if adapter.config.get_id not in ("static-enikshay-episode", "static-np-migration-3-episode"):
```

Comment:
nitpick I think it'd be better to use `if adapter.config.get_id in (...): return` to make it more obvious